Integrations/IronDefense/IronDefense.yml

```diff
@@ -182,15 +310,679 @@ script:
+      description: True if the event activity was blacklisted, false otherwise.
+      type: Boolean
+    - contextPath: IronDefense.Event.src_ip
+      description: The source IP associated with this event.
+      type: String
+    - contextPath: IronDefense.Event.dst_ip
+      description: The destination IP associated with this event.
+      type: String
+    - contextPath: IronDefense.Event.dst_port
+      description: The destination port associated with this event.
+      type: Number
+    - contextPath: IronDefense.Event.ppp_domains
+      description: Any domains associated with this event.
+      type: String
+    - contextPath: IronDefense.Event.primary_app_protocol
+      description: The primary application protocol associated with this event.
+      type: String
+    - contextPath: IronDefense.Event.secondary_app_protocol
+      description: The secondary application protocol associated with this event.
+      type: String
+    - contextPath: IronDefense.Event.bytes_in
+      description: The byte count of incoming traffic for this event.
+      type: Number
+    - contextPath: IronDefense.Event.bytes_out
+      description: The byte count of outgoing traffic for this event.
+      type: Number
+    - contextPath: IronDefense.Event.total_bytes
+      description: The byte count of the total traffic (in either direction) for this
+        event.
+      type: Number
+    - contextPath: IronDefense.Event.url
+      description: The related URL for this event, if applicable.
+      type: String
+    - contextPath: IronDefense.Event.raw_data_formats
+      description: The list of distinct raw data formats for this event.
+      type: String
+    - contextPath: IronDefense.Event.src_entity_attribute
+      description: The source entity attribute related to this event, if one could
+        be determined.
+      type: String
+    - contextPath: IronDefense.Event.src_entity_attribute_type
+      description: The source entity attribute type related to this event, if one
+        could be determined.
+      type: String
+    - contextPath: IronDefense.Event.dst_entity_attribute
+      description: The destination entity attribute related to this event, if one
+        could be determined.
+      type: String
+    - contextPath: IronDefense.Event.dst_entity_attribute_type
+      description: The destination entity attribute type related to this event, if
+        one could be determined.
+      type: String
+    - contextPath: IronDefense.Event.vue_url
+      description: The url for displaying the event within IronVUE.
+      type: String
+    - contextPath: IronDefense.Event.context.name
+      description: The name of the context table.
+      type: String
+    - contextPath: IronDefense.Event.context.columns
+      description: Column values for the context table.
+      type: String
+  - arguments:
+    - default: false
+      description: The list of Alert IDs to filter by.
+      isArray: false
```

Comment:
if it's a list than this one should be true

Reply:
For these we're not taking it as an array. We're  taking a CSV string and splitting it.